calc/HamiltonianOne.cpp

```diff
@@ -407,7 +407,7 @@ void HamiltonianOne::build() {
 #pragma omp parallel for schedule(static, 1)
 
     // Loop through steps
-    for (size_t step = 0; step < nSteps; ++step) {
+    for (int step = 0; step < static_cast<int>(nSteps); ++step) {
```

Comment:
No, unfortunately, Visual Studio 2015 does fully support only the OpenMP 2.0 specifications.